Automattic/fresh-data
# Update babel to 7

Wow, this is a lot of lines, but this is all build stuff and no real code changes exist in this PR.
It will need some testing though.

Also, known issue: The example apps use an older version of `react-scripts` with a security vulnerability. But it's only when they're being served locally. That will be a topic for another PR to update them all to CRA2.

(Edit: I created #104 to track the above issue)

## To Test:

1. `npm install`
2. `npm run bootstrap` (this will take a while)
3. `npm run build`
4. `npm run lint`
5. `npm test`
6. Test each example app by:
  a. `cd examples/<app>`
  b. `npm install`
  c. `npm start`
  c. Browser tab should open, try out app and make sure it works.
